content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
-
-![Screenshot: final configuration of tasks within a workspace](./images/run-tasks-final-workspace-configuration.png)
+Your run tasks are now configured. You can return to the settings page to configure or remove tasks in your workspace.
+    ![Screenshot: final configuration of tasks within a workspace](./images/run-tasks-final-workspace-configuration.png)
 
 
 ## Understanding Run Tasks Within a Run
 
-Workspaces with run tasks configured have an additional run stage called [Pre-Apply](../run/states.html). This stage is where your run tasks will execute during a run, after the plan stage and also the cost estimation and policy check stages, if applicable.
+Run tasks execute between the [plan](../run/states.html#2-the-plan-stage) and [apply](../run/states.html#6-the-apply-stage) stage of a [Terraform run](../run/index.html). Once all run tasks executions have completed, the run will end based on the most restrictive enforcement level that has been configured in each associated run tasks.
 
-The Pre-Apply stage will always end with the most restrictive status of the tasks configured to run. For example, if a mandatory task fails and an advisory task succeeds, the stage will fail and the run will error. If an advisory task fails but a mandatory task succeeds, the stage will succeed and the run will proceed to the apply stage.
+For example, if a mandatory task fails and an advisory task succeeds, the run will fail. If an advisory task fails, but a mandatory task succeeds, the run will succeed and proceed to the apply stage. Regardless of the exit status of a task, the status and any related message data will be displayed in the UI.
 
-Regardless of the exit status of a task, the status and any related message will be displayed in the UI.
-
-Here is an example of a run that failed due to a mandatory task.
+Here is an example of a run that failed due to a mandatory run task.
 
 ![Screenshot: a run with failed tasks](./images/run-tasks-run-failed.png)
 
 And here is an example of a run that succeeded.
 
-![Screenshot: a run with failed tasks](./images/run-tasks-run-success.png)
+![Screenshot: a run with passed tasks](./images/run-tasks-run-success.png)
+
+## Removing a Run Task from a Workspace
+
+1. Navigate to the desired workspace, then Click **Run Tasks** from the **Settings** menu.
 
-## Deleting a Task from a Workspace
+2. Click the ellipses (...) on the associated run task, and then Click **Remove**.
 
-1. Navigate to the desired workspace, then select "Tasks" from the "Settings" menu.
+    ![Screenshot: removing a run task from a workspace](./images/run-tasks-workspace-remove.png)
 
-2. Click Remove next to the task you wish to delete
 
+## Deleting a Run Task
 
-## Deleting a Task Event Hook
+1. Navigate to **Settings** and **Run Tasks**
 
-1. Navigate to Settings and Task event hooks
+2. On the run task you want to delete, Click the ellipses and then Click **Edit**
```

Comment:
```suggestion
2. On the run task you want to delete, Click the ellipses (...) and then Click **Edit**
```

This introduces the parenthesis here too, as I feel it probably isn't enough to do it once. This suggestion doesn't change `Click` to lowercase nor change it ultimately to `select` though as I'd prefer to keep those as separate discussions.

Reply:
In another style guide that I referred to, they suggest that you show a visual representation of the term in the first instance and then just use the word in subsequent step. I am not attached to my approach, just what makes sense.

Reply:
Yeah, I guess that works then. Have to keep on top of those first instance in documentation moving forward then. Maybe there's a Markdown hacky way to do it.